src/main/java/fr/openent/competences/service/impl/DefaultNoteService.java

```diff
@@ -2432,7 +2430,17 @@ public void getTotaleDatasReleve(final JsonObject params, final Long idPeriode,
                                         null, true, annual,
                                         compNotesEvent -> formate(compNotesFuture, compNotesEvent));
 
+                                Promise<Boolean> isAvgSkillpromise = Promise.promise() ;
+                                structureOptionsService.getIsAverageSkills(idEtablissement, event -> {
+                                    if(event.isRight())
+                                        isAvgSkillpromise.complete(event.right().getValue().getBoolean("is_average_skills"));
+                                    else
+                                        isAvgSkillpromise.fail(event.left().getValue());
+                                });
+
+
```

Comment:
Tu pourrais dans DefaultStructureOptions ajouter la méthode `isAverageSkills` qui reprendrait la méthode pour seulement retourner true comme on s'aperçoit qu'il est très souvent utilisé : 

DefaultStructureOptions : 
```
@Override
    public Future<Boolean> isAverageSkills(String structureId) {
        Promise<Boolean> promise = Promise.promise();
        getIsAverageSkills(structureId, event -> {
            if(event.isRight())
                promise.complete(event.right().getValue().getBoolean(Field.IS_AVERAGE_SKILLS)); // pour "is_average_skills"
            else
                promise.fail(event.left().getValue());
        });
        return promise.future();
    }
```

Et donc par la suite faire : 

```suggestion
        Future<Boolean> isAvgSkillpromise = structureOptionsService.isAverageSkills(idEtablissement);
                listFutures.add(isAvgSkillpromise);
```